tcms/rpc/tests/test_environment.py

```diff
@@ -14,3 +18,64 @@ def verify_api_with_permission(self):
     def verify_api_without_permission(self):
         with self.assertRaisesRegex(ProtocolError, "403 Forbidden"):
             self.rpc_client.Environment.filter(None)
+
+
+@override_settings(LANGUAGE_CODE="en")
+class EnvironmentCreate(APITestCase):
+    def test_environment_create_with_no_required_fields(self):
+        values = {}
+        with self.assertRaisesRegex(
+            XmlRPCFault, "name.*This field is required."
+        ):
+            self.rpc_client.Environment.create(values)
+
+        values["description"] = "Description"
+        with self.assertRaisesRegex(XmlRPCFault, "name.*This field is required"):
+            self.rpc_client.Environment.create(values)
+
+    def test_environment_duplicate(self):
+        values = {"name": "E7", "description": "Description"}
+        self.rpc_client.Environment.create(values)
+        with self.assertRaisesRegex(XmlRPCFault, "name.*Environment with this Name already exists"):
+            self.rpc_client.Environment.create(values)
+
+    def test_environment_create_with_chinese(self):
```

Comment:
Test method not needed, we're already testing different locales, there is no need for them to be tested for every API method. Remove it.

Reply:
Done